scripts/globals/mobskills/flame_blast.lua

```diff
@@ -0,0 +1,25 @@
+---------------------------------------------
+--  Flame Blast
+--
+--  Description: Deals fire damage to enemies within a wide area of effect.
+--  Type: Magical
+--  Utsusemi/Blink absorb: Ignores shadows
+--  Range: 30'
+--  Notes: Used only by KS99 Wyrm. Only use in a dedicated flying skill set.
```

Comment:
Can specify that this version is the grounded version.

Just to clarify, Flame Blast is supposed to be magical and not a breath attack, right? The skill check is okay (ie: can use regardless of where target is)?

Reply:
I think you're getting these mixed up. There is no grounded version Flame_Blast ;)
One is single target regular attack (_alt)
and the other (this one) is the TP AoE skill. Yes it's supposed to be magic damage, not breath.

Reply:
Also on a further note: During flight all Wyrms (should) face their target, so directions do not apply and don't need to be checked for indeed.

Reply:
Oh, so this version is the one it'll actually display the message for, got it! Carry on!